jwst/datamodels/tests/test_schema.py

```diff
@@ -376,12 +372,10 @@ def test_table_array_convert():
     Test that structured arrays are converted when necessary, and
     reused as views when not.
     """
-    from jwst.datamodels import util
 
     table_schema = {
         "allOf": [
-            mschema.load_schema(os.path.join(URL_PREFIX, "image.schema"),
-                resolver=asdf.AsdfFile().resolver,
+            mschema.load_schema("http://stsci.edu/schemas/jwst_datamodel/core.schema",
```

Comment:
Should  `image.schema` be tested here?

Reply:
A base `DataModel` was being tested before

https://github.com/spacetelescope/jwst/pull/5433/files#diff-c83baf50e80f78e21b7a376fe4208a97ea9421e27f8ca0b3a12121820c8ce588L417

And I've just changed it to a `JwstDataModel` which only depends on the core schema.

So I think this is a bit clearer and consistent, especially since nothing specific to `ImageModel` is used in these tests.